server/channels/app/platform/helper_test.go

```diff
@@ -148,6 +151,10 @@ func SetupWithCluster(tb testing.TB, cluster einterfaces.ClusterInterface) *Test
 }
 
 func setupTestHelper(dbStore store.Store, dbSettings *model.SqlSettings, enterprise bool, includeCacheLayer bool, tb testing.TB, options ...Option) *TestHelper {
+	return setupTestHelperWithConfigModifiers(dbStore, dbSettings, enterprise, includeCacheLayer, tb, options)
+}
+
+func setupTestHelperWithConfigModifiers(dbStore store.Store, dbSettings *model.SqlSettings, enterprise bool, includeCacheLayer bool, tb testing.TB, options []Option, configModifiers ...ConfigModifier) *TestHelper {
```

Comment:
this is basically cause `EnableWebHubChannelIteration` can only be applied before the Hub starts